.cirrus.yml

```
@@ -16,11 +23,8 @@ build_task:
     cpu: 8
     memory: 4G
   env:
-    # Possible values for ARTIFACTORY_DEPLOY_REPO: sonarsource-private-qa, sonarsource-public-qa
-    ARTIFACTORY_DEPLOY_REPO: FIXME
-    ARTIFACTORY_DEPLOY_USERNAME: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-qa-deployer username]
-    ARTIFACTORY_DEPLOY_PASSWORD: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-qa-deployer access_token]
-    DEPLOY_PULL_REQUEST: "true"
+    SONAR_TOKEN: FIXME
+    SONAR_HOST_URL: FIXME
```

It should be possible to use Next now. First create the Project from SPEED
```suggestion
    SONAR_TOKEN: VAULT[development/kv/data/next data.token]
    SONAR_HOST_URL: VAULT[development/kv/data/next data.url]
```

---

I will solve this in another PR as I have to create the next project first